lua/lazyvim/plugins/extras/lang/php.lua

```diff
@@ -48,7 +48,7 @@ return {
     optional = true,
```

Comment:
`php-debug-adapter` doesn't seem to be installed anywhere. I did a repo serach and could only find the one occurrence in this spec's `ops` function. We could add it as a spec so it's only installed if the dap extra is installed.

```suggestion
    optional = true,
    specs = {
        {
            "mason-org/mason.nvim",
            opts = {
              ensure_installed = {
                "php-debug-adapter",
              },
            },
        },
    },
```